packages/grafana-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -4,7 +4,7 @@ import { Popover } from './Popover';
 import { PopoverController, UsingPopperProps } from './PopoverController';
 
 export interface TooltipProps extends UsingPopperProps {
-  theme?: 'info' | 'error';
+  theme?: 'info' | 'error' | 'code';
```

Comment:
I woke up with a thought today, that maybe instead of being very explicit with `code` name, we could have `info-alt`?

Reply:
Hm, yeah maybe. Just need to add some documentation about that

Reply:
There is one already: https://github.com/grafana/grafana/blob/master/contribute/style-guides/styling.md#styling-with-theme